tests/index.js

```diff
@@ -9,6 +9,7 @@ require('./spec/filer.spec');
 // Filer.FileSystem.*
 require('./spec/filer.filesystem.spec');
 require('./spec/fs.spec');
+require('./spec/fs.access');
```

Comment:
Can you rename your file `tests/spec/fs.access.spec.js` to match everything else and update this line here to also include `.spec` at the end?